docs/installation/upgrading.rst

```diff
@@ -1,31 +1,97 @@
 .. _upgrading-ksql:
 
-Upgrading KSQL
-==============
+Upgrading ksqlDB
+================
 
-Upgrade one KSQL server at a time (i.e. rolling restart). The remaining KSQL servers should have sufficient spare
-capacity to take over temporarily for unavailable, restarting servers.
+Upgrade one server at a time in a "rolling restart". The remaining servers
+should have sufficient spare capacity to take over temporarily for unavailable,
+restarting servers.
+
+For general guidance on upgrading, see
+[Upgrade ksqlDB](https://docs.ksqldb.io/en/latest/operate-and-deploy/installation/upgrading/).
+
+Upgrading to ksqlDB 5.5 from KSQL 5.4
+-------------------------------------
+
+.. important::
+
+   The upgrade from KSQL 5.4 to ksqlDB 5.5 is not a rolling restart. You must
+   shut down all KSQL instances and then start up all ksqlDB instances, so there
+   will be downtime.
+
+#. Prepare a list of SQL statements for migration. One way is to dump the command
+   topic.
+
+   ```bash
+   ./bin/kafka-console-consumer --bootstrap-server localhost:9092 --topic _confluent-ksql-default__command_topic --from-beginning | jq ".statement" 
+   ```
+
+#. List your application's streams, tables, and types. Don't use the EXTENDED
+   option. Go through each list, and find the the last CREATE statement for each
+   stream/table/type name. Order the CREATE statements so they show the correct
+   dependencies.
+   
+#. Make other compatibility-related changes, as shown in the
+   `upgrade guide <https://docs.ksqldb.io/en/latest/operate-and-deploy/installation/upgrading/#how-to-upgrade>`__.
+
+#. Stop the ``ksql`` service.
+
+#. Uninstall the ksql 5.4 DEB or RPM package.
+
+#. Install the ksqldb 5.5 DEB or RPM package.
```

Comment:
for RPM - you can just use `yum swap` (not sure if we want to mention that here though because I guess what you have would work)